docs/community-blog-posts.md

```diff
@@ -0,0 +1,7 @@
+blog
+
+https://docs.google.com/document/d/1mRdAwySlpQCGC3a4jK9fp2gj1RGTopBZOdsyFUZFWjI/edit?usp=sharing
```

Comment:
Can you add the link like `[some text](hyperlink here)`

Reply:
yes , doing that.